games-puzzle/tanglet/tanglet-1.6.1.1.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit qmake-utils xdg
+
+DESCRIPTION="Single player word finding game based on Boggle"
+HOMEPAGE="https://gottcode.org/tanglet/"
+SRC_URI="https://gottcode.org/${PN}/${P}-src.tar.bz2"
```

Comment:
Trivial but `${PN}` be nicer plaintexted
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Plaintexting_.24.7BPN.7D_in_SRC_URI